src/styles/main.scss

```
@@ -0,0 +1,57 @@
+.face {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(215, 246, 255);
+  &__center,
+  &__second,
+  &__minute {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  &__center {
+    z-index: 3;
+    width: 30px;
+    height: 30px;
+    background-color: black;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+  }
+  &__second {
+    z-index: 2;
+    width: 6px;
+    height: 170px;
+    background-color: orange;
+    border-radius: 3px;
+    transform-origin: bottom center;
+    animation: seconds 60s steps(60) infinite;
+  }
+  &__minute {
+    z-index: 2;
+    width: 15px;
+    height: 200px;
+    background-color: brown;
+    border-radius: 7px;
+    transform-origin: bottom center;
+    animation: seconds 3600s steps(60) infinite;
+  }
+  @media (max-width: 500px) {
+    &__minute {
+      height: 40vw;
+    }
+    &__second {
+      height: 34vw;
+    }
+  }
+}
+
+@keyframes seconds {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
```

```suggestion
@keyframes seconds {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
```
should work the same without this prop, since it's not changed